### PR TITLE
Added the IAC response team

### PR DESCRIPTION
**Pull request by Geevies**
This PR splits the ERT file up into a bunch of files, to edit them later much easier.

1 Doctor (Lead), intended to do advanced medical stuff to heal people.
2 Paramedics, intended to bring injured to the doctor. (Can likely also do advanced medical stuff, but the doctor should be the primary one to do so)
1 Bodyguard, intended to stay with and defend in the IAC team. (Also trained in medical procedures, but not super advanced.)

image

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by BoryaTheSlayer** [comment]
Very cool

File: code/datums/outfits/ert/iac.dm [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		var/obj/item/clothing/shoes/footwraps = new(src)
+		H.equip_to_slot_if_possible(footwraps, slot_shoes)
+
+/datum/outfit/admin/ert/iac/bodyguard
+	name = "IAC Bodyguard"
+
+	accessory = /obj/item/clothing/accessory/storage/webbing
+	accessory_contents = list(/obj/item/reagent_containers/hypospray/autoinjector/norepinephrine = 2, /obj/item/reagent_containers/hypospray/autoinjector/survival = 1)
+	suit = /obj/item/clothing/suit/armor/tactical
+	suit_store = /obj/item/gun/projectile/automatic/rifle
```
Comment (fernerr) [inline code/datums/outfits/ert/iac.dm]:
```suggestion
	suit_store = /obj/item/gun/projectile/automatic/rifle/sts35
```

Comment (fernerr) [reply code/datums/outfits/ert/iac.dm]: automatic/rifle is not intended to be used, sts35 is what you're looking for.

**Comment by fernerr** [comment]
I think the splitting of the ert teams file into separate ones should be noted in the pr description.

**Comment by fernerr** [comment]
Tis' a shame we can't have ert teams spawn custom equipment to specified landmarks for example, these guys could use a field surgery table kit, an iv, some blood bags and maybe a medical mech.

File: code/datums/outfits/ert/iac.dm (hunk 1 above)
Comment (Alberyk) [reply code/datums/outfits/ert/iac.dm]: I am also not sure if carrying around large ballistic weapons even fits the iac, I would recommend a laser carbine at most.

Comment (fernerr) [reply code/datums/outfits/ert/iac.dm]: An energy rifle might be a good compromise, for a bit more longevity + the ability to stun.

File: code/datums/outfits/ert/iac.dm [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	back = /obj/item/storage/backpack/satchel_med
+	accessory = /obj/item/clothing/accessory/storage/white_vest
+	accessory_contents = list(/obj/item/reagent_containers/hypospray/cmo = 1, /obj/item/storage/pill_bottle/dexalin_plus = 1, /obj/item/storage/pill_bottle/tramadol = 1)
+	id = /obj/item/card/id/distress/iac
+
+	l_ear = /obj/item/device/radio/headset/distress
+
+	backpack_contents = list(
+		/obj/item/storage/box/survival = 1,
+		/obj/item/storage/firstaid/regular = 2,
```
Comment (NonQueueingMatt) [inline code/datums/outfits/ert/iac.dm]: I'd make this an adv medkit tbh.

File: code/datums/outfits/ert/iac.dm [hunk 3]
```diff
@@ -0,0 +1,87 @@
+/datum/outfit/admin/ert/iac
+	name = "IAC Responder"
```
Comment (Arrow768) [inline code/datums/outfits/ert/iac.dm]:
```suggestion
	name = "IAC Doctor"
```

File: code/datums/outfits/ert/iac.dm [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+	belt_contents = list(
+		/obj/item/ammo_magazine/c45x = 2,
+		/obj/item/device/flash = 1,
+		/obj/item/handcuffs/ziptie = 2,
+		/obj/item/melee/baton = 1,
+		/obj/item/device/flashlight/flare = 1
+	)
+
+/datum/outfit/admin/ert/iac/paramedic
+	name = "IAC Paramedic Responder"
```
Comment (Arrow768) [inline code/datums/outfits/ert/iac.dm]:
```suggestion
	name = "IAC Paramedic"
```

File: code/game/objects/items/weapons/cards_ids.dm [hunk 5]
```diff
@@ -467,6 +467,15 @@ var/const/NO_EMAG_ACT = -50
 	access = list(access_legion, access_maint_tunnels, access_external_airlocks, access_security, access_engine, access_engine_equip, access_medical, access_research, access_atmospherics, access_medical_equip)
 	..()
 
+/obj/item/card/id/distress/iac
+	name = "\improper Interstellar Aid Corps ID"
```
Comment (Arrow768) [inline code/game/objects/items/weapons/cards_ids.dm]: Would be nice to get some special ID cards / description text for it

File: code/modules/ghostroles/spawner/human/responseteams/iac.dm [hunk 6]
```diff
@@ -0,0 +1,22 @@
+/datum/ghostspawner/human/ert/iac
+	name = "IAC Responder"
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/iac.dm]:
```suggestion
	name = "IAC Doctor"
```

File: code/modules/ghostroles/spawner/human/responseteams/iac.dm [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+	short_name = "iacresponder"
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/iac.dm]:
```suggestion
	short_name = "iacdoctor"
```

File: code/modules/ghostroles/spawner/human/responseteams/iac.dm [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+/datum/ghostspawner/human/ert/iac/bodyguard
+	name = "IAC Bodyguard"
+	short_name = "iacbodyguard"
+	max_count = 1
+	desc = "A highly trained bodyguard. Sticks close to the medics while they work."
+	outfit = /datum/outfit/admin/ert/iac/bodyguard
+
+/datum/ghostspawner/human/ert/iac/paramedic
+	name = "IAC Paramedic Responder"
```
Comment (Arrow768) [inline code/modules/ghostroles/spawner/human/responseteams/iac.dm]:
```suggestion
	name = "IAC Paramedic"
```

File: code/datums/outfits/ert/iac.dm (hunk 2 above)
Comment (Geevies) [reply code/datums/outfits/ert/iac.dm]: regular's better than adv with brainmed i think

File: code/game/objects/items/weapons/cards_ids.dm (hunk 5 above)
Comment (Geevies) [reply code/game/objects/items/weapons/cards_ids.dm]: probably

**Pull request merged**